go/client/wallet_api_listen_display.go

```diff
@@ -43,21 +46,33 @@ func newWalletNotificationDisplay(g *libkb.GlobalContext) *walletNotificationDis
 	return &walletNotificationDisplay{
 		baseNotificationDisplay: newBaseNotificationDisplay(g),
 		cli:                     cli,
+		deduper:                 make(map[string]bool),
```

Comment:
is there any concern that for the life of the command this will just increase in size?

Reply:
i was thinking about that. i figured the data structure is likely so small, even if you're sending/receiving a ridiculous amount of payments on a very long-lived process, this would probably not be the first thing to break. but i definitely do not know that for sure. if you think it's worth using a different data structure that only keeps the more recent 100 or so transactions, i can look into that.

Reply:
let's just go with this.